UPDATE.md

```diff
@@ -38,8 +37,11 @@ in the composer.json of your project.
 
 ### Pre-requirements for the media update
 After that the following steps should be done for the update:
-* Add drupal/media_entity ^2.0 to your composer.json
-* Add drupal/video_embed_field ^2.0 to your composer.json
+
+```
+composer require drupal/media_entity:^2.0 drupal/media_entity_image drupal/video_embed_field
+```
+
 * Make sure that you use the "Media in core" branch for all your
 media_* modules. (For the media modules in Thunder, we take care of that)
 * Make sure that all your media_entity related code is moved to media.
```

Comment:
This is not so understandable. It would be better to write something like:
`Make sure that all your code that uses media_entity API is modified to use core media API.`